.github/workflows/ci.yml

```diff
@@ -113,19 +114,22 @@ jobs:
           node-version: "17.9"
 
         - name: Node.js 18.x
-          node-version: "18.14"
+          node-version: "18.20.4"
```

Comment:
```suggestion
          node-version: "18"
```

What if we stick only to the major? That way we don't need to worry updating each time.

Reply:
I like that option